docs/self-managed/reference-architecture/kubernetes.md

```diff
@@ -0,0 +1,287 @@
+
+A multi-namespace setup allows you to duplicate and run multiple Orchestration clusters based on different use cases and requirements, while Web Modeler and Console remains independent, and does not need to be scaled the same way. The same applies to Identity as a central entity coordinating access management.
+
+The **Orchestration cluster** namespace, as outlined in the [architecture diagram](#orchestration-cluster), consists of the following components:
+
+- [Zeebe Brokers](/components/zeebe/technical-concepts/architecture.md#brokers)
+- [Zeebe Gateway](/self-managed/zeebe-deployment/zeebe-gateway/zeebe-gateway-overview.md)
+- [Operate](/components/operate/operate-introduction.md)
+- [Tasklist](/components/tasklist/introduction-to-tasklist.md)
+- [Optimize]($optimize$/components/what-is-optimize)
+- [Connectors](/components/connectors/introduction.md)
+
+The **Web Modeler and Console** namespace, as outlined in the [architecture diagram](#web-modeler-and-console), consists of the following components:
+
+- Web Modeler
+- [Console](/self-managed/console-deployment/overview.md)
+- [Identity](/self-managed/identity/what-is-identity.md)
+  - Keycloak, an external dependency of Identity
+
+The **Orchestration cluster** should be configured to use the central Identity of the **Web Modeler and Console** namespace.
+
+## Requirements
+
+This guide focuses on a single-region application, but can be adapted for a multi-region setup once you understand the basics of a single region.
+
+For more details on multi-region configurations, especially dual-region setups, refer to the [dedicated guide](/self-managed/concepts/multi-region/dual-region.md).
+
+### Infrastructure
+
+We recommend using an officially [certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/#benefits) distribution.
+
+Camunda 8 is not tied to a specific Kubernetes version. To simplify deployment, we provide a [Helm chart](/self-managed/setup/install.md) for easy installation on Kubernetes. The latest Helm chart is typically compatible with Kubernetes' [official support cycle](https://kubernetes.io/releases/).
+
+#### Minimum cluster requirements
+
+The following are suggested minimum requirements. Sizing depends heavily on your specific use cases and workload. See [sizing your environment](/components/best-practices/architecture/sizing-your-environment.md) and [Zeebe resource planning](/self-managed/zeebe-deployment/operations/resource-planning.md) for additional resources, conduct benchmarking to determine your exact requirements.
+
+- 4 Kubernetes Nodes
+  - Modern CPU: 4 Cores
+  - Memory: 16 GiB
+- Persistent Volumes
+  - 1,000 IOPS
+  - 32 GiB size
+  - _avoid burstable disk types_
+
+#### Networking
+
+While networking is largely managed through services and Load Balancers, understanding port usage can be useful - especially if explicit whitelisting is required within a private network.
```

Comment:
I wanted to rephrase this but wasn't entirely sure:
```suggestion
Networking is largely managed through services and Load Balancers. The following describes networking requirements, including port usage, which may be necessary if explicit whitelisting is required within a private network.
```

Reply:
Not 100% sure with the wording either, let's go with your version for now.

Reply:
backported it to 8.7 👍